src/components/dialogs/affiliation-error.vue

```diff
@@ -1,19 +1,36 @@
 <template>
-  <v-dialog v-model="getAffiliationErrorModalVisible" max-width="60%" persistent>
-    <v-card class="notify-dialog">
-      <v-card-title class="flex-column">
-        <v-icon large color="error" class="my-4">mdi-alert</v-icon>
-        <span>Error Affiliating Name Request</span>
+  <v-dialog :value="getAffiliationErrorModalValue" max-width="45rem" persistent>
```

Comment:
I updated the max width so the dialog looks like this on a wide screen:

![image](https://user-images.githubusercontent.com/10002889/187489605-b997df08-cbda-42fd-be3f-e0d5677128e1.png)

And like this on a narrow screen:

![image](https://user-images.githubusercontent.com/10002889/187489784-e4c91ef4-baab-4520-80b6-4166d8870c65.png)